app/views/subnets/_fields.html.erb

```diff
@@ -3,7 +3,15 @@
 
 <%= subnet_type_f f %>
 <%= text_f f, :network, :label => _("Network Address") %>
-<%= text_f f, :cidr, :label => _("Network Prefix"), :help_inline => _("Prefix length for this subnet, e.g. 24") %>
+<%= text_f f, :cidr, :label => _("Network Prefix"),
+                 :label_help => _("While it is possible to define the same DHCP range in Foreman, it's usually good practice to select a range from outside the pool defined in the installer, but still in the subnet.<br/>" +
+                                    "For the example above, it is recommended to define the DHCP range from 10.0.0.1 to 10.0.0.99 in the Foreman UI which gives the following IP address distribution:<br/>" +
+                                    "<li>10.0.0.1 - 10.0.0.99 - addresses reserved during bare-metal provisioning by Foreman</li></ul>" +
+                                    "<li>10.0.0.100 - 10.0.200 - addresses for dynamic clients in the subnet (discovered hosts, unmanaged hosts)</li></ul><br/>" +
+                                    "More about this in manual: " + documentation_button('3.2.3InstallationScenarios')
```

Comment:
can we use a link instead of the docs button?

Reply:
Definitely, I have changed it.